src/ios/CDVInAppBrowser.m

```diff
@@ -1137,8 +1228,11 @@ - (id)init
 		self.hidden = NO;
 		self.disallowoverscroll = NO;
 		
+		NSString *closeString = @"Close";
+		self.closebuttoncaption = closeString;
```

Comment:
I don't think this is necessary. Keep self.closebuttoncaption is already set to nil in earlier lines